docs/user/alerting/pre-configured-connectors.asciidoc

```diff
@@ -49,12 +53,35 @@ The following example shows a valid configuration 2 out-of-the box connector.
 
 [NOTE]
 ==============================================
-Sensitive properties, such as passwords, can also be stored in the {kib} keystore.
+Sensitive properties, such as passwords, can also be stored in the <<creating-keystore, {kib} keystore>>.
 ==============================================
 
+[float]
+[[preconfigured-action-type-example]]
+=== Creating a preconfigured action type
+
+In the `kibana.yml` file:
```

Comment:
Suggest removing these lines to make it consistent with the section on preconfigured connectors.  Also the text below the code box explains about xpack.actions.enabled.

Lines to remove:

In the kibana.yml file:

Exclude the action type from xpack.actions.enabledActionTypes.
Add all its preconfigured connectors.